### PR TITLE
Update latency report in latency.c

### PR DESCRIPTION
**Pull request by Wenwen-Chen**
This patch try to correct the latency report.
1. Rename Redis to Valkey.
2. Remove redundant Dave dialog, and refine the output message. Refer to #426
@hwware pls help to review the patch

### CODE REVIEW

File: src/latency.c [hunk 1]
```diff
@@ -358,18 +356,17 @@ sds createLatencyReport(void) {
...
-        report = sdscat(report, "Dave, no latency spike was observed during the lifetime of this Redis instance, not "
-                                "in the slightest bit. I honestly think you ought to sit down calmly, take a stress "
-                                "pill, and think things over.\n");
+        report = sdscat(report, "No latency spike was observed during the lifetime of this Valkey instance, not "
+                                "in the slightest bit.\n");
     } else if (eventnum > 0 && advices == 0) {
         report =
-            sdscat(report, "\nWhile there are latency events logged, I'm not able to suggest any easy fix. Please use "
-                           "the Redis community to get some help, providing this report in your help request.\n");
+            sdscat(report, "\nThere are latency events logged, they are not easy to fix. Please get "
```
Comment (zuiderkwast) [inline src/latency.c]:
```suggestion
            sdscat(report, "\nThere are latency events logged that are not easy to fix. Please get "
```

Either like this or use two separate sentences: There are latency events logged. They are not easy to fix."

**Review by zuiderkwast — APPROVED** [review]
Thanks for completing this!

File: src/latency.c [hunk 2]
```diff
@@ -477,18 +474,18 @@ sds createLatencyReport(void) {
...
                                     "Cluster large keys migrations (RESTORE command), may create sudden memory "
                                     "pressure forcing the server to block trying to evict keys. \n");
         }
 
         if (advise_disable_thp) {
             report = sdscat(report, "- I detected a non zero amount of anonymous huge pages used by your process. This "
                                     "creates very serious latency events in different conditions, especially when "
-                                    "Redis is persisting on disk. To disable THP support use the command 'echo never > "
+                                    "Valkey is persisting on disk. To disable THP support use the command 'echo never > "
                                     "/sys/kernel/mm/transparent_hugepage/enabled', make sure to also add it into "
```
Comment (zuiderkwast) [inline src/latency.c]: This line is too long (121 characters) so clang-format wants to change the formatting. It's better to fix it like this IMO:

```suggestion
                                    "Valkey is persisting on disk. To disable THP support use the command 'echo never "
                                    "> /sys/kernel/mm/transparent_hugepage/enabled', make sure to also add it into "
```

Then please run `clang-format -i src/latency.c` or fix manually what it reported in the clang-format CI job log.

**Comment by hwware** [comment]
@Wenwen-Chen Please install clang-format on your local, the process is: 

1. install clang-format
2. git add your_update_file
3. run "git clang-format"

File: src/latency.c (hunk 1 above)
Comment (Wenwen-Chen) [reply src/latency.c]: Thanks for your suggestion. I will fix it.

**Review by zuiderkwast — APPROVED** [review]

**Pull request merged**